spec/unit/blockscore/question_set_spec.rb

```diff
@@ -54,17 +54,25 @@ module BlockScore
       let(:incorrect_answers) { QuestionSetHelper.incorrect_answers(question_set.questions) }
 
       context 'correct answers' do
-        before { question_set.score(correct_answers) }
-
-        its(:score) { should eq 100.0 }
-        it { expect(question_set.score(correct_answers)).to eq 100.0 }
+        it { expect(question_set.score(correct_answers)).to equal 100.0 }
```

Comment:
Instance of https://github.com/BlockScore/blockscore-ruby/pull/62/files#r52250764